charts/cray-certmanager/values.yaml

```diff
@@ -6,12 +6,19 @@
 # https://github.com/jetstack/cert-manager/pull/1828
 curl:
   image:
-    repository: artifactory.algol60.net/csm-docker/stable/docker.io/curlimages/curl
+    registry: artifactory.algol60.net
```

Comment:
Do you find that you need to override the registry in one or two cases, but not in every? If we need to override everything every time then would a global.registry option work better? plus we can more easily change it at a platform level if it's a standard value across charts.

Reply:
Was more if i was setting it one way I was going to to it the same way for all, i'd ideally like to kill this chart entirely and just install cert-manager directly.

Reply:
More to be consistent really, if I set it up one way I'd do the same in each location. I'd rather we do globals in an umbrella chart and pass that into subcharts in a unified way but thats a bit out of scope for this.